### PR TITLE
Feature/27/impl add photo step

### LINKED ISSUE #27: (SP: 1) Implement Add photo step
**User Stories** #57, #58
## Description
Implement Add photo step for both student and tutor steppers, use existing drag-and-drop, add text and input for file

## Design 

Image

## Acceptance Criteria
- [ ] When user clicks on input it should open window to choose file from computer for loading
- [ ] When user drag file on drag-and-drop it should load file
- [ ] When file is loaded it should be shown on the left block

### LINKED ISSUE #29: (SP: 3) Implement loading and validation of the file on Add photo step
**User Stories** #57, #58
**Blocked by** #102 

## Description
Implement loading and validation of the file on Add photo step

## Design 

Image

## Acceptance Criteria
- [ ] When file is loaded it should send file to server
- [ ] When file is not `.png`, `.jpg` or `.jpeg` it should validate and throw an error

### PR DESCRIPTION
**Pull request by geersann**
###Issue
Open https://github.com/WebUI-Project-Based-10-nr/Client/issues/27 Implement Add Photo Step

###Description

1. Added Page type
2. Added Page styles
3. Added a button with the function of selecting a file from the Explorer
          3.1 Added validation from type
          3.2 Added validation from file size
          3.3 Added save photo in state
4. Added dragging file
5. Rendered added photo in left block

###Assignees
assignees: @geersann

### CODE REVIEW

File: src/containers/tutor-home-page/general-info-step/GeneralInfoStep.jsx [hunk 1]
```diff
@@ -59,7 +112,7 @@ const GeneralInfoStep = ({ btnsBox }) => {
             <AppTextField
               fullWidth
               label={t('common.labels.firstName')}
-              onBlur={handleBlur('firstName')}
+              onBlur={() => {}}
```
Comment (petro97) [inline src/containers/tutor-home-page/general-info-step/GeneralInfoStep.jsx]: Why did you remove the handleBlur function here?

File: src/containers/tutor-home-page/general-info-step/GeneralInfoStep.jsx [hunk 2]
```diff
@@ -7,31 +7,84 @@ import { styles } from '~/containers/tutor-home-page/general-info-step/GeneralIn
...
+    const fetchCities = (country) => {
+      const countryCode = countryCodes[country]
+
+      if (countryCode) {
+        axios
+          .get(
+            `${CITIES_API_URL}?formatted=true&country=${countryCode}&maxRows=10&username=geer_sann`
+          )
+          .then((response) => {
+            console.log(response.data)
```
Comment (petro97) [inline src/containers/tutor-home-page/general-info-step/GeneralInfoStep.jsx]: just remove this console.log

File: src/containers/tutor-home-page/general-info-step/GeneralInfoStep.jsx (hunk 1 above)
Comment (geersann) [reply src/containers/tutor-home-page/general-info-step/GeneralInfoStep.jsx]: My bad, it’s file incorrect to this ticket and push, it doesn’t matter

**Review by Yurii-04 — APPROVED** [review]

**Review by Annaivv — APPROVED** [review]

**Review by hanus-kostiantyn — APPROVED** [review]

**Review by dimaP2341 — APPROVED** [review]

**Review by nrevutska — APPROVED** [review]
